.gitmodules

```diff
@@ -4,4 +4,4 @@
 	branch = develop
 [submodule "test/tests"]
 	path = test/jsontests
-	url = https://github.com/ethereum/tests.git
+	url = https://github.com/jwasinger/tests.git
```

Comment:
I was modifying the submodules as per @pirapira 's document: https://github.com/ethereum/tests/wiki/Creating-a-GeneralStateTest-(Yoichi's-attempt,-rough-record) .  Obviously we don't want those changes actually included in the repo tho.

I forgot to specify the branch for the tests:  invalid-ops.  It's on jwasinger/tests.  In the future I'll just open branches directly on the tests/cpp-ethereum repos.

I also reverted the submodules change.  See if the new stBadOpcode tests are working now.